From 2778c30d4de251f0e0e7951f63f60de4d65c7c25 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Sun, 1 Aug 2021 05:18:07 +0530
Subject: [PATCH] [EHL-CIV] Media,Video performance fix

CTS Media and Video Performanc range fix for WW30_02

Change-Id: I3904b789d8cea446041223b5e7d15910c253ce7f
Tracked-On: OAM-97958
Signed-off-by: vdanix <vishwanathx.dani@intel.com>

diff --git a/groups/codecs/configurable/media_codecs_performance_ehl.xml b/groups/codecs/configurable/media_codecs_performance_ehl.xml
index 2b1932f..851ca76 100644
--- a/groups/codecs/configurable/media_codecs_performance_ehl.xml
+++ b/groups/codecs/configurable/media_codecs_performance_ehl.xml
@@ -31,7 +31,7 @@
             <Limit name="measured-frame-rate-640x360" range="98-152" />
             <Limit name="measured-frame-rate-720x480" range="170-190" />
             <Limit name="measured-frame-rate-1280x720" range="70-120" />
-            <Limit name="measured-frame-rate-1920x1080" range="40-70" />
+            <Limit name="measured-frame-rate-1920x1080" range="18-36" />
         </MediaCodec>
         <MediaCodec name="c2.android.mpeg4.decoder" type="video/mp4v-es" update="true">
             <Limit name="measured-frame-rate-176x144" range="164-271" />
@@ -49,17 +49,17 @@
             <Limit name="measured-frame-rate-1920x1080" range="50-100" />
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_vd.h264" type="video/avc" update="true">
-            <Limit name="measured-frame-rate-320x240" range="1500-2000" />
-            <Limit name="measured-frame-rate-720x480" range="1000-2000" />
+            <Limit name="measured-frame-rate-320x240" range="300-450" />
+	    <Limit name="measured-frame-rate-720x480" range="400-550" />
             <Limit name="measured-frame-rate-1280x720" range="460-900" />
             <Limit name="measured-frame-rate-1920x1088" range="320-390" />
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_vd.h265" type="video/hevc" update="true">
-            <Limit name="measured-frame-rate-352x288" range="1200-1250" />
-            <Limit name="measured-frame-rate-640x360" range="1000-1400" />
-            <Limit name="measured-frame-rate-720x480" range="820-1020" />
-            <Limit name="measured-frame-rate-1280x720" range="540-580" />
-            <Limit name="measured-frame-rate-1920x1080" range="430-470" />
+            <Limit name="measured-frame-rate-352x288" range="320-430" />
+            <Limit name="measured-frame-rate-640x360" range="380-450" />
+            <Limit name="measured-frame-rate-720x480" range="260-380" />
+            <Limit name="measured-frame-rate-1280x720" range="220-290" />
+            <Limit name="measured-frame-rate-1920x1080" range="190-220" />
             <Limit name="measured-frame-rate-3840x2160" range="120-130" />
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_vd.vp8" type="video/x-vnd.on2.vp8" update="true">
@@ -69,9 +69,9 @@
             <Limit name="measured-frame-rate-1920x1080" range="100-330" />
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_vd.vp9" type="video/x-vnd.on2.vp9" update="true">
-            <Limit name="measured-frame-rate-320x180" range="1300-1800" />
+            <Limit name="measured-frame-rate-320x180" range="385-425" />
             <Limit name="measured-frame-rate-640x360" range="500-1300" />
-            <Limit name="measured-frame-rate-1280x720" range="560-590" />
+            <Limit name="measured-frame-rate-1280x720" range="200-230" />
             <Limit name="measured-frame-rate-1920x1080" range="350-420" />
             <Limit name="measured-frame-rate-3840x2160" range="100-130" />
         </MediaCodec>
@@ -79,9 +79,9 @@
     <Encoders>
         <MediaCodec name="c2.android.avc.encoder" type="video/avc" update="true">
             <Limit name="measured-frame-rate-320x240" range="97-143" />
-            <Limit name="measured-frame-rate-720x480" range="150-170" />
-            <Limit name="measured-frame-rate-1280x720" range="65-75" />
-            <Limit name="measured-frame-rate-1920x1080" range="30-35" />
+            <Limit name="measured-frame-rate-720x480" range="25-45" />
+            <Limit name="measured-frame-rate-1280x720" range="15-35" />
+            <Limit name="measured-frame-rate-1920x1080" range="10-15" />
         </MediaCodec>
         <MediaCodec name="c2.android.hevc.encoder" type="video/hevc" update="true">
             <Limit name="measured-frame-rate-320x240" range="25-55" />
--
2.32.0

